src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,16 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    public static class ColorWheel {
+        public static final double[] RED = {0, 0, 0};
+        public static final double[] GREEN = {0, 0, 0};
+        public static final double[] BLUE = {0, 0, 0};
+        public static final double[] YELLOW = {0, 0, 0};
+
+        public static final int MAX_HALF_SPINS = 6;//Amount of times the color wheel must spin in half spins.
```

Comment:
Why MAX, 3 spins it's the minimal amount of spins you need to do in order to succeed the task